lib/Tooling/Refactor/ImplementDeclaredMethods.cpp

```diff
@@ -408,21 +426,15 @@ ImplementDeclaredObjCMethodsOperation::runInImplementationAST(
   std::string MethodString;
   llvm::raw_string_ostream OS(MethodString);
 
-  PrintingPolicy PP = Context.getPrintingPolicy();
-  PP.PolishForDeclaration = true;
-  PP.SuppressStrongLifetime = true;
-  PP.SuppressLifetimeQualifiers = true;
-  PP.SuppressUnwrittenScope = true;
-  for (const ObjCMethodDecl *MD : SelectedMethods) {
+  assert(MethodDeclarations.size() >= SelectedMethods.size() &&
+         "less declarations than selected methods?");
+  for (const auto &I : llvm::enumerate(SelectedMethods)) {
+    indexer::Indexed<const ObjCMethodDecl *> Decl = I.value();
     // Skip methods that are already defined.
```

Comment:
Yes, but in practice this won't normally happen. Because of how this is implemented, the indexer will either find out whether all the queried methods are defined, or the continuation will receive an empty vector of methods.